.gitignore

```diff
@@ -37,4 +37,4 @@ build/
 
 # Unified Installer: ignore env files that devs may end up keeping around (but whitelist the example)
 cluster-install/*.env
-cluster-install/install-example-config.env
+!cluster-install/install-example-config.env
```

Comment:
The intent was to make this an exception but forgot the `!` when I ported over the gitignore changes.